cromshell

```diff
@@ -427,6 +456,42 @@ function populateLastWorkflowId()
   WORKFLOW_ID=$( tail -n1 ${CROMWELL_SUBMISSIONS_FILE} | awk '{print $3}' )
 }
 
+function alias_workflow() 
+{
+	local alias_name=${2}
+
+	# Check that we have an alias and workflow defined:
+	if [[ ${#alias_name} -eq 0 ]] || [[ ${#2} -eq 0 ]] ; then
```

Comment:
didn't you set $2 -> alias_name?  do you mean to use $1 instead here?

Reply:
Ah!  Yes.

In practice the call to `populateWorkflowIdAndServerUrl` was catching the case where the workflow was undefined, but I'll make sure to fix it here too.